src/app/api/ai/route.ts

```
@@ -57,12 +81,14 @@ export async function POST(req: Request) {
         case "mock":
             const validatedMockPayload = MockPayloadSchema.safeParse(mock);
 
-            if (!validatedMockPayload.success) return Response.json({ error: "Bad Request" }, { status: 400 });
+            if (!validatedMockPayload.success)
+                return Response.json({ error: "Bad Request" }, { status: 400 });
 
-            const { maxMarks, semester, branch, subject, topics } = validatedMockPayload.data;
+            const { maxMarks, semester, branch, subject, topics } =
+                validatedMockPayload.data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)